Client/JavaScripts/LongPress.js

```diff
@@ -0,0 +1,57 @@
+function findTagsElementAtPoint(x,y) {
+    var tags = [];
+    var e = document.elementFromPoint(x,y);
+    while (e) {
+        if (e.tagName) {
+            tags.push("" + e.tagName);
+        }
+        e = e.parentNode;
+    }
+    return tags;
+}
+
+function findImageSrcLinkAtPoint(x,y) {
+    var  imageLink = {};
+    e = document.elementFromPoint(x,y);
+    while (e && ! e.src) {
+        e = e.parentNode;
+    }
+
+    if (e && e.src) {
+        imageLink["imageSrc"] = e.src
+        var r = e.getClientRects()[0];
+        imageLink["location"] = '{{' + r.left + ',' + r.top + '},{' + r.width + ',' + r.height + '}} ';
+    }
+   return imageLink;
+}
+
+function findHrefLinkAtPoint(x,y) {
+    var  hrefLink = {};
+    e = document.elementFromPoint(x,y);
+    while (e && ! e.href) {
+        e = e.parentNode;
+    }
+
+    if (e && e.href) {
+        hrefLink["hrefLink"] = e.href
+        var r = e.getClientRects()[0];
+        hrefLink["location"] = '{{' + r.left + ',' + r.top + '},{' + r.width + ',' + r.height + '}} ';
+    }
+    return hrefLink;
+}
+
+function findElementsAtPoint(x,y) {
+    var tags = findTagsElementAtPoint(x,y);
```

Comment:
I don't really see the value of doing this. Why not just unconditionally call findHrefLinkAtPoint/findImageSrcLinkAtPoint?